docs/examples.rst

```diff
@@ -7,80 +7,85 @@ Basics
-
-    [forcing_vars]
-    prec = Prec
-    t_max = Tmax
-    t_min = Tmin
-
-    [state_vars]
-    prec = prec
-    t_max = t_max
-    t_min = t_min
-
-    [domain_vars]
-    lat = lat
-    lon = lon
-    mask = mask
-    elev = elev
-
-This is a minimal configuration file for MetSim, and contains 3 sections.  The
-first section, ``[MetSim]`` describes some basic settings such as the locations
-of data and parameters used in calculations.  For a complete description of the
-input format see the `configuration <configuration.rst>`_ page.  The key things to note in this section
-are the options specified under the ``# Input specification`` and ``# Output
-specification`` comment headers.  The ``forcing`` and ``domain`` options refer
-to the two types of required input, and the ``in_format`` and ``out_format``
-options tell MetSim how they should be treated.
-
-The second and third sections (``[forcing_vars]`` and ``[state_vars]``) describe the variables in the
-datasets provided in the ``forcing`` and ``state`` options of the first section.
-The left side of the assignment is the name of the variable given
-in the ``forcing`` dataset, while the right hand side is the
-name the variable should be given within MetSim.  Note that the
-variables shown here are the minimum required set to run the
-forcing generation. The names given on the right hand side are
-also important to name correctly, as they are referenced internally.
-If you are unsure what variable names are used internally see the
-`configuration <configuration.rst>`_ page for a full breakdown.
+    # Overall configuration, specification of parameters and input/output
+    # paths goes in the "MetSim" section
+    MetSim:
+        # Time step in minutes
+        time_step: 30
+        # Forcings begin here (year-month-day)
+        start: 1950-1-1
+        # Forcings end at this date (year-month-day)
+        stop: 1950-1-31
+        # Input and output directories
+        forcing: './metsim/data/test.nc'
+        domain: './metsim/data/tiny_domain.nc'
+        state: './metsim/data/state_nc.nc'
+        forcing_fmt: 'netcdf'
+        in_format: 'netcdf'
+        out_dir: './results'
+        out_prefix: 'yaml_output'
+        prec_type: 'triangle'
+        utc_offset: True
+
+    out_vars:
+        temp:
+            out_name: 'airtemp'
+            units: 'K'
+        prec:
+            out_name: 'pptrate'
+            units: 'mm/s'
+        shortwave:
+            out_name: 'SWradAtm'
+        spec_humid:
+            out_name: 'spechum'
+        air_pressure:
+            out_name: 'airpres'
+            units: 'kPa'
+        wind:
+            out_name: 'windspd'
+
+    chunks:
+        lat: 3
+        lon: 3
+
+    forcing_vars:
+        # Format is metsim_name: input_name
+        prec  : 'Prec'
+        t_max : 'Tmax'
+        t_min : 'Tmin'
+
+    state_vars:
+        # Format is metsim_name: input_name
+        prec  : 'prec'
+        t_max : 't_max'
+        t_min : 't_min'
```

Comment:
This is a bit confusing. Why would the `state_vars` input names be different than the `forcing_vars` (I mean - you could I guess, but is that really the common use case?)

Reply:
I'll update this so that they're the same. I don't really know if it's common, but I do have at least a couple test cases where this is true, which is probably more of an artifact of MetSim changing through time rather than anything else.